README.md

```diff
@@ -1,11 +1,14 @@
 SCUBA  [![Build Status](https://github.com/JonathonReinhart/scuba/actions/workflows/build-test.yml/badge.svg)](https://github.com/JonathonReinhart/scuba/actions/workflows/build-test.yml) [![codecov.io](https://codecov.io/github/JonathonReinhart/scuba/coverage.svg?branch=master)](https://codecov.io/github/JonathonReinhart/scuba?branch=master) [![PyPI](https://img.shields.io/pypi/v/scuba.svg)](https://pypi.python.org/pypi/scuba) [![Docs](https://readthedocs.org/projects/scuba/badge/?version=latest)](https://scuba.readthedocs.io/)
 -----
+![SCUBA](https://user-images.githubusercontent.com/5616068/195471179-ef1b04e7-0d45-4f81-9460-e1af8a150199.png)
 
-Simple Container-Utilizing Build Apparatus
+**SCUBA** is the **S**imple **C**ontainer-**U**tilizing **B**uild **A**pparatus
 
 Scuba makes it easier to use Docker containers in everyday development. It is
-intended to be used by developers with `make` or `scons`-based build systems,
-where the entire build environment is encapsulated in a Docker container.
+intended to allow a developer to commit an environment setup where the entire
```

Comment:
I like changing this to make it more generic. Can we drop the "intended to" and make it just "allows"? I.e. "It allows a developer..."

Reply:
Consider it done